tests/timescaledb/copydb.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+set -x
+set -e
+
+# This script expects the following environment variables to be set:
+#
+#  - PGCOPYDB_SOURCE_PGURI
+#  - PGCOPYDB_TARGET_PGURI
+#  - PGCOPYDB_TABLE_JOBS
+#  - PGCOPYDB_INDEX_JOBS
+
+# make sure source and target databases are ready
+pgcopydb ping --source ${PGCOPYDB_SOURCE_PGURI} --target ${PGCOPYDB_TARGET_PGURI}
+
+psql -a ${PGCOPYDB_SOURCE_PGURI} <<EOF
+create role tsdb NOSUPERUSER CREATEDB NOCREATEROLE LOGIN PASSWORD '0wn3d';
+create database tsdb owner tsdb;
+EOF
+
+# copying roles needs superuser
+# and we use the postgres database here still
+pgcopydb copy roles --source ${PGCOPYDB_SOURCE_PGURI} --target ${PGCOPYDB_TARGET_PGURI}
+
+# now create the tsdb database on the target, owned by new role tsdb
+psql ${PGCOPYDB_TARGET_PGURI} <<EOF
+create database tsdb owner tsdb;
+EOF
+
+psql -o /tmp/c.out -d ${PGCOPYDB_SOURCE_PGURI} -1 -f /usr/src/pgcopydb/rides.sql
+
+copy="\COPY rides FROM nyc_data_rides.10k.csv CSV"
+psql -d ${PGCOPYDB_SOURCE_PGURI} -1 -c "${copy}"
+
+# take a snapshot using role tsdb on source database
+coproc ( pgcopydb snapshot --debug )
+
+sleep 1
+
+# copy the extensions separately, needs superuser (both on source and target)
+pgcopydb list extensions
+
+# now clone with superuser privileges, seems to be required for timescaledb
+pgcopydb clone --skip-extensions \
```

Comment:
I don't see anywhere the extension being copied., adding --skip-extension wouldn't migrate TimescaleDB catalog tables.